go.mod

```diff
@@ -14,14 +14,25 @@ require (
 	github.com/go-openapi/jsonpointer v0.19.5 // indirect
 	github.com/go-openapi/swag v0.21.1 // indirect
 	github.com/google/uuid v1.3.0 // indirect
-	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/invopop/yaml v0.1.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
+	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51 // indirect
 	github.com/mailru/easyjson v0.7.7 // indirect
+	github.com/mattn/go-colorable v0.1.13 // indirect
+	github.com/mattn/go-isatty v0.0.16 // indirect
+	github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b // indirect
 	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826 // indirect
 	github.com/perimeterx/marshmallow v1.1.4 // indirect
-	github.com/spf13/pflag v1.0.5 // indirect
-	golang.org/x/sys v0.4.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+require (
+	github.com/Masterminds/semver v1.5.0
+	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/pkg/errors v0.9.1
+	github.com/spf13/pflag v1.0.5 // indirect
+	golang.org/x/sys v0.4.0 // indirect
+	golang.org/x/text v0.6.0
+	gopkg.in/AlecAivazis/survey.v1 v1.8.8
```

Comment:
I'm skeptical about using third-party libraries, why do we need a lib for a simple command prompt?

Reply:
survey.v1 lib : https://github.com/openshift/backplane-cli/blob/7d72524c95ccb96608c23b473f2368576fb7cbce/internal/upgrade/upgrade.go#L151

Reply:
Can't we do it the normal way?
ref: https://github.com/openshift/pagerduty-short-circuiter/blob/d7ed6a6796845680e2ee5a0b5ad424ffde46cc57/cmd/kite/teams/cmd.go#L93

Reply:
Can we do it without using any libs as it's just a user prompt? :)

Reply:
All good .. Remove all unwanted dependencies on `upgrade` mod and use  `bufio.Reader` 
https://github.com/openshift/backplane-cli/blob/72c5601cb5a465492fb8b7853874b92e7b0d325a/internal/upgrade/upgrade.go#L149